src/lib/markbind/src/patches/htmlparser2.js

```diff
@@ -95,20 +99,35 @@ var i = 0,
     IN_NUMERIC_ENTITY         = i++,
     IN_HEX_ENTITY             = i++, //X
 
-    j = 0,
+	SPECIAL_NONE              = 0,
 
-    SPECIAL_NONE              = j++,
-    SPECIAL_SCRIPT            = j++,
-    SPECIAL_STYLE             = j++;
+	/*
+	 Non parser-state constants
+	 */
+
+	// processSpecialFunctions, processSpecialClosingTagCharacter
+	HAS_MATCHING              = -1,
+	NO_MATCH                  = -2,
+	HAS_MATCHED               = -3;
+
+function whitespace(c) {
+	return c === " " || c === "\n" || c === "\t" || c === "\f" || c === "\r";
```

Comment:
Should we use a regex whitespace check instead? I'm just a little concerned about doing an explicit check because there are so many different ways to get whitespace :P

Reply:
It's a method from htmlparser2